spec/services/pii/signing_key_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe Pii::SigningKey do
+  describe '#new' do
+    it 'automatically loads the server-wide signing key' do
+      expect(subject.rsa_key).to be_a OpenSSL::PKey::RSA
+    end
+  end
+
+  describe '#sign' do
+    it 'signs text with private key' do
+      signature = subject.sign('foo')
+
+      expect(signature).to be_a String
+    end
+  end
+
+  describe '#verify' do
+    it 'confirms signature' do
+      signature = subject.sign('foo')
+
+      expect(subject.verify('foo', signature)).to eq true
+      expect(subject.verify('bar', signature)).to eq false
+    end
+
+    it 'throws exception on invalid signature' do
+      signature = subject.sign('foo')
+
+      expect do
+        subject.verify('foo', signature + '!')
```

Comment:
it took me a few reads to understand why this failed and the example above did not. The reason is what I was not seeing the `+ '!'`

I think if we put `signature + '!'` in a separate var and used that here it would be easier to understand on a quick scan